### PR TITLE
Fix coverage report generation for the top-level project

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # git master
 
+* [486](https://github.com/Eyescale/CMake/pull/486):
+  Fix coverage report generation for non-subprojects
```
Comment (hernando) [inline CHANGES.md]: Indeed, you have solved it for the subproject case. Don't you mean "for non-superprojects"?

Comment (rdumusc) [inline CHANGES.md]: This is from Daniel's commit but it is correct. It was already working for the subproject case (i.e. a project which is a subproject of viz/stable for instance). Now it is fixed (with a small limitation) for projects which are not a subproject but are their own superproject (thus non-subprojects). Nomenclature is a bit tricky for sure :-)

Comment (hernando) [inline CHANGES.md]: We have to agree on the nomenclature, because what you call the subproject case (viz/stable) for me it's the superproject case.

Comment (rdumusc) [inline CHANGES.md]: Or: fix coverage report generation for the top-level project?

Comment (hernando) [inline CHANGES.md]: I prefer that one.

**Comment by rdumusc** [comment]
updated

**Comment by hernando** [comment]
+1

**Pull request merged**